be/src/service/point_query_executor.cpp

```diff
@@ -380,6 +396,17 @@ Status PointQueryExecutor::_lookup_row_key() {
         specified_rowsets = _tablet->get_rowset_by_ids(nullptr);
     }
     std::vector<std::unique_ptr<SegmentCacheHandle>> segment_caches(specified_rowsets.size());
+    // init segment_cache
```

Comment:
Why add this logic?

Reply:
in order to trace the time of loading segments